src/getting-started/deployment.md

```diff
@@ -53,7 +53,7 @@ COPY resources /app/resources
 # Not needed if you are using embed configuration
 COPY config.production.json ./config.production.json
 
-RUN useradd -r -U go-exec -M -d /app -s /bin/false
+RUN adduser -S go-exec 
```

Comment:
```suggestion
RUN adduser --system --group --no-create-home --home /app --shell /bin/false go-exec 
```

I think it is better to use the long-form flags to make it more explicit. Also, to make it identical, can you please add the other options above?

Reply:
I updated it. I added --system and --home, but system users by default don't get a shell, and on alpine (busybox) it's /sbin/nologin.